### PR TITLE
Share backup

### PR DESCRIPTION
**Pull request by xkevin190**
-added the functionality of making a backup file
- Share the backup file with multiple applications
- account and database restoration with backup file.

### CODE REVIEW

File: android/app/google-services.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
         }
       },
       "oauth_client": [
```
Comment (luisan00) [inline android/app/google-services.json]: Is this related to the G.Drive permission?

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -8,7 +8,6 @@
     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE" />
     <!-- < Only if you're using GCM or localNotificationSchedule() > -->
 
-    <uses-permission android:name="android.permission.VIBRATE" />
```
Comment (luisan00) [inline android/app/src/main/AndroidManifest.xml]: Not necessary?

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2229,6 +2223,9 @@
 					"$(SRCROOT)..\node_modules\neact-native-soundRNSound",
 					"$(SRCROOT)..\node_modules@react-native-communitysliderios",
 					"$(SRCROOT)..\node_modules\neact-native-locale-detectorRNI18n",
+					"$(SRCROOT)/../node_modules/react-native-document-picker/ios/RNDocumentPicker",
+					"$(SRCROOT)/../node_modules/@react-native-community/google-signin/ios/RNGoogleSignin",
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: This is not necessary

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2410,23 +2389,14 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Here the same I think

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -2475,23 +2445,14 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Other Google stuff:)

File: yarn.lock [hunk 6]
```diff
@@ -6853,6 +6863,11 @@ react-native-gesture-handler@^1.0.12:
     invariant "^2.2.2"
     prop-types "^15.5.10"
 
+react-native-google-signin@^2.1.1:
```
Comment (luisan00) [inline yarn.lock]: Google stuff :lollipop:

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -2345,23 +2333,14 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Google stuff :lollipop:

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -2279,23 +2276,14 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Google stuff? :koala:

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 9]
```diff
@@ -2186,6 +2177,9 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Google stuff? :apple:

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 10]
```diff
@@ -2127,24 +2127,15 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Google stuff? :mag:

File: ios/LochaMeshChat.xcodeproj/project.pbxproj [hunk 11]
```diff
@@ -2069,24 +2078,15 @@
...
```
Comment (luisan00) [inline ios/LochaMeshChat.xcodeproj/project.pbxproj]: Google stuff! :pick:

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (xkevin190) [reply android/app/src/main/AndroidManifest.xml]: was repeated

File: android/app/google-services.json (hunk 1 above)
Comment (xkevin190) [reply android/app/google-services.json]: Google libraries are needed for the notification library, here only sha1 is added to the application

**Pull request merged**